helper.py

```diff
@@ -28,32 +39,46 @@ def delete_files(path):
         else:
             import warnings
 
-            warnings.warn(f"{filename!r} does not seem to be a file, did not remove it.")
+            warnings.warn(
+                f"{filename!r} does not seem to be a file, did not remove it."
+            )
 
 
-def optimade_api_version():
-    """Print OPTIMADE API version supported in currently installed optimade package as input for shell array"""
+def optimade_api_version() -> None:
+    """Print OPTIMADE API version supported in currently installed optimade package as
+    input for shell array."""
     try:
         from optimade import __api_version__
     except ImportError:
-        exit("optimade MUST be installed to run 'api-version'")
+        sys.exit("optimade MUST be installed to run 'api-version'")
 
     versions = [
-        __api_version__.split('-')[0].split('+')[0].split('.')[0],
-        '.'.join(__api_version__.split('-')[0].split('+')[0].split('.')[:2]),
-        '.'.join(__api_version__.split('-')[0].split('+')[0].split('.')[:3]),
+        __api_version__.split("-", maxsplit=1)[0]
+        .split("+", maxsplit=1)[0]
+        .split(".", maxsplit=1)[0],
+        ".".join(
+            __api_version__.split("-", maxsplit=1)[0]
+            .split("+", maxsplit=1)[0]
+            .split(".")[:2]
+        ),
+        ".".join(
+            __api_version__.split("-", maxsplit=1)[0]
+            .split("+", maxsplit=1)[0]
+            .split(".")[:3]
+        ),
     ]
```

Comment:
I think this code is inefficient, could you not do:
```suggestion
    split_version = __api_version__.split("-", maxsplit=1)[0].split("+", maxsplit=1)[0].split(".")
    versions = [split_version[0], ".".join(split_version[:2]), ".".join(split_version[:3])
```

Reply:
Yeah - this is a good idea. I'm also a bit confused about this variable in general, I need to see the output to convince myself that everything isn't actually repeated here.
Great catch.